### DISCUSSION TITLE
Removing models from the UI

### DISCUSSION THREAD

**Discussion by ArcadiaBaes** [Mustard2/MustardUI#183] [Help] [answered]
Hello. I have added outfits from different models to another, all of them using mustards UI. Now I would like to remove models im not using from the UI, but i dont know how I can do that. Should I edit the plugin with python?

![image](https://github.com/user-attachments/assets/aaaf634d-23e8-4286-b301-1e92ec5d5990)

**Comment by Mustard2** [comment]
Hi! Maybe I'm not understanding the question.
Do you want to delete one of the models from the scene? If yes, just completely remove the armature (you might need to switch to Blender data in the Outliner panel)

**Comment by ArcadiaBaes** [comment]
my model is the first rachel amber. i added some clothes from other models (they use your UI) and remove their armatures, but they are still in the UI. I dont know how to remove their name, as they are useless. somebody told me ill have to edit the UI

**Comment by Mustard2** [comment, marked as answer]
Ah no need to modify it, you just need to be careful to delete the correct data. :)

Click on the armature and check the name of the data of that armature (the Armature data - green one icon, not the Object one - yellow icon - see the image)

![sc_20241221_21230_935](https://github.com/user-attachments/assets/bbf6468f-fe53-431c-b354-958e326fb702)

then remove it from the Outliner -> Blender file

![sc_20241221_212053_180](https://github.com/user-attachments/assets/0ebb514d-ee7d-4d6d-847f-cd705fd9e6bc)

This should remove the armature from the list immediatly.

This is due to a Blender bug related to double linked data.

**Reply by ArcadiaBaes** [reply]
As i have not imported the second models armature to my file, i have to create a new empty armature and... it works! thank you

**Comment by ArcadiaBaes** [comment]
i found another way: clean up, purge unused datas

**Reply by Mustard2** [reply]
Yes, and this is exactly what the new MustardUI X button near the models do. :)
